source/__tests__/integration/jestPuppeteerHelper.js

```diff
@@ -0,0 +1,55 @@
+async function expectSelTextContentToMatch(sel, expected) {
+  await page.waitForSelector(sel)
+  const textContent = await page.$eval(sel, e => e.textContent)
+  return expect(textContent).toMatch(expected)
+}
+async function expectSelTextContentNotToMatch(sel, expected) {
+  const textContent = await page.$eval(sel, e => e.textContent)
```

Comment:
is the assumption that the element will exist, but that we're looking for it to not have the text in `expected`?  or is it ok if the element doesn't exist at all?  if the former, what about an `await page.waitForSelector(sel)` at the start of the function?  (if the latter, disregard this comment)

Reply:
I think it's "selector may exist, but want it NOT to have the value".   I've been buried in these !#$! tests for so long ... I just don't wanna fuss anymore.  They aren't great tests to begin with, and they're not well organized and we're basically making as few changes to the profile editor as possible because it's in OLD angular blah blah.

Reply:
fine by me 😄

Reply:
this is clarified in PR #218